sdmetrics/reports/utils.py

```diff
@@ -479,6 +479,133 @@ def get_column_pair_plot(real_data, synthetic_data, column_names, metadata):
         return make_mixed_column_pair_plot(real_data, synthetic_data)
 
 
+def get_cardinality(parent_table, child_table, parent_primary_key, child_foreign_key):
```

Comment:
can we make this private for now?

Reply:
Yes I was thinking of this first, but then I saw that we don't do it in this file. Done in [76fcf86](https://github.com/sdv-dev/SDMetrics/pull/384/commits/76fcf862286df85acac9ee2977216ac5c72f5a30)